### ISSUE TITLE
Inconsistent dataset of Moving_MNIST in the paper and code? 

### ISSUE THREAD

**Issue by ILoveStudying** [thuml/predrnn-pytorch#26]
Dear authors, thanks for sharing your code and datasets. The dataset volume described in the paper is fixed, with 10000 sequences for the training set, 3000 sequences for the validation set and 5000 sequences for the test set.
However,  the dataset provided in the GitHub is different, with 10000 sequences for the training set, 2000 sequences for the validation set and 3000 sequences for the test set.
So, could you pl tell me the difference? Thanks!

**Comment by wuhaixu2016** [comment]
Hi, this code repo is correct. We will rephrase the paper soon.
